### PR TITLE
Update QUEST.tsv [Lines 1-500]

### PR DESCRIPTION
**Pull request by ghost**
Resending pull request #655 to make sure the changes show.

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -300,203 +300,203 @@ QUEST_20150317_000299	No. Try again.
...
-QUEST_20150317_000495	Find it. The road will open when it is proven that you are the true Revelator.
-QUEST_20150317_000496	What makes you walk towards the light in a place full of darkness?
-xQUEST_20150317_000497	The harder you try to find this, the stronger you become, but if you don't find it, you will become weak. {Nl}What is this?
+xQUEST_20150317_000494	I have prepared five dilemmas so that our enemy can't get their hands on it. {nl}But neither of you can pass it if you are not prepared.
+xQUEST_20150317_000495	Find it. The road will open when it has been proven that you are the true Revelator.
+xQUEST_20150317_000496	What makes you walk towards the light in a place full of darkness?
+xQUEST_20150317_000497	The harder you try to find it, the stronger you become, but if you don't find it, you become weak. {nl} What is it?
 QUEST_20150317_000498	Aide de camp Selina
-xQUEST_20150317_000499	Our days to stay self-sufficient is growing. {nl}We even joke about plowing a farm ourselves,  {nl}but it's not becoming a joke anymore.
```
Comment (vtange) [inline QUEST.tsv:499]: I think the old version of the line is more accurate. It's more about the guys staying around longer than they had originally thought.

File: QUEST.tsv [hunk 2]
```diff
@@ -10,286 +10,286 @@ xQUEST_20150317_000009	Please select a starting zone.
...
-xQUEST_20150317_000031	I charge 100 Silver. {nl}That will cover operations, labor, and what have you.
+xQUEST_20150317_000031	I charge 100 silver. {nl}That will cover operations, labor and what have you.
 xQUEST_20150317_000032	Hey, wait a minute. {nl}You know you need more than that…
 xQUEST_20150317_000033	That's no good! {nl}I can't work for nothing, even for a Revelator.
 xQUEST_20150317_000034	Wandering Merchant
-xQUEST_20150317_000035	Like a dandelion's seeds, I am merely one who follows the wind. {nl}Since we've met in such a curious way, perhaps you'll spare a moment to view my curiosities?
+xQUEST_20150317_000035	Like a dandelion's seeds, I am merely one who follows the wind. {nl}Since we've met in such a curious way, perhaps you could spare a moment to view my curiosities?
 xQUEST_20150317_000036	Kristina
-xQUEST_20150317_000037	Welcome! {nl}You're looking for a travel companion, right? {nl}You won't find a better place for partners than here!
+xQUEST_20150317_000037	Welcome! {nl}You're looking for a travel companion, right? {nl}Then you've come to the right place!
```
Comment (vtange) [inline QUEST.tsv]: Big change in meaning without the word "partner". I would go with something like "This is the right place to find a partner!"

File: QUEST.tsv [hunk 3]
```diff
@@ -300,203 +300,203 @@ QUEST_20150317_000299	No. Try again.
...
-QUEST_20150317_000305	Well, at least the divine district is still left. {nl}That's all because of the blessing of the goddess.
-QUEST_20150317_000306	I am just an ordinary old man in Fedimian. {nl}If you want to know more, bring me some potato dumplings…
-xQUEST_20150317_000307	The food at Fedimian is too awful. {nl}Before I die, is it possible to try all delicious food in the world?
-QUEST_20150317_000308	This is the divine district in Fedimian. {nl}May the goddess bless you in the future.
-xQUEST_20150317_000309	Right now, the commerce district is closed for reconstruction.{nl}Please do not enter the district.
+xQUEST_20150317_000303	Fedimian became like this four years ago…{nl}It was such a beautiful and vibrant city. {nl}Fortunately, the divine district, where the central statue of the Goddess is located, is still okay.
+xQUEST_20150317_000304	Fedimian was originally divided into two districts, the commerce district and the divine district. {nl}But an incident left the commerce district completely destroyed…
```
Comment (vtange) [inline QUEST.tsv]: Lost the detail that the incident was 4 years ago. This line makes it sound like it was just another incident

File: QUEST.tsv [hunk 4]
```diff
@@ -300,203 +300,203 @@ QUEST_20150317_000299	No. Try again.
...
+xQUEST_20150317_000327	If one soldier breaks a rule, then it may endanger the entire military. {nl}That's why we have to be very strict.
+xQUEST_20150317_000328	I guess I'll admit it now. You, the Revelator, pursue the dream of the Goddess. {nl}It's not going to be easy, but I believe you'll do well.
+xQUEST_20150317_000329	Are you the Revelator? Good job, {nl}but I'm not quite sure I can trust you.
 QUEST_20150317_000330	Private Personnel
-xQUEST_20150317_000331	To survive in this chaotic world, sometimes you need to do use some dirty tricks. {nl}You shouldn't feel bad about it.
-QUEST_20150317_000332	You should run for it if you feel something is not being done quickly. {nl}That's why I became a soldier.
-QUEST_20150317_000333	I heard many stories about heroes who fight for justice in my childhood, {nl}but I never thought one of those stories can be real…{nl}Now I think heroes may exist.
+xQUEST_20150317_000331	To survive in this chaotic world, you'll sometimes need to use some dirty tricks. {nl}You shouldn't feel bad about it.
+xQUEST_20150317_000332	If you want to get a job done quick, then do it yourself. {nl}That's why I became a soldier.
```
Comment (vtange) [inline QUEST.tsv:332]: Major meaning change. Would check original Korean.

**Comment by CasparHansen** [comment]
Hi Senhart, It looks like you forked this file before the last batch of PRs was merged in. Please merge your changes with the updates made since then to make sure things don't go backwards.

**Comment by ghost** [comment]
Closed for resubmission in line with the new QUEST.tsv.

**Pull request closed without merging**